src/TargetDescription.cc

```diff
@@ -0,0 +1,128 @@
+#include "TargetDescription.h"
+#include "GdbServerConnection.h"
+#include "kernel_abi.h"
+#include <sstream>
```

Comment:
Order #includes like our other files: first `TargetDescription.h`, then the C++ library headers, then the rr headers.